sdk/boms/azure-sdk-bom/pom.xml

```diff
@@ -5,7 +5,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.azure</groupId>
   <artifactId>azure-sdk-bom</artifactId>
-  <version>1.0.3-beta.1</version>
```

Comment:
For BOM - when do we choose to beta vs non-beta?

Reply:
The BOM will be in beta status between releases but when being released it changes to GA

Reply:
Sorry for being unclear - do we want to GA every 3 months and then be in beta in between? Do we expect some feedback on the BOM files while we are in beta?

Reply:
Current plan is releasing a new `azure-sdk-bom` version at least once every three months. There are no plans on having betas in-between as the `azure-sdk-bom` provides no functionality but just manages dependencies.

Reply:
Yup that is why I was curious why we chose beta the last time :)